ckanext/extrafields/tests/test_create_dataset.py

```diff
@@ -52,13 +52,22 @@ def teardown_class(cls):
         ckan.plugins.unload('extrafields')
 
     def test_package_create_with_minimum_values(self):
-        '''If dataset is given name it is created.
+        '''If dataset is given it's basic fields it is created.
         '''
         dataset = helpers.call_action(
             'package_create',
             name='package-name',
-            short_description='short description text'
+            title_translated={
```

Comment:
Why is the attribute "title_translated" now? Would it not work as "title", or is that reserved as a default?

Reply:
Short version: that's how it works.

Longer version: Any CKAN core fields should not be modified directly by fluent. Any that should be translated (with some exceptions) are flagged with a _translated suffix and a new field is created that stores the translations.  Effectively "title" becomes "title_translated['en']".

Reply:
right, I guess short_description is not core, so that could be overridden.

Reply:
yup